frontend/src2/dashboard/DashboardItem.vue

```diff
@@ -22,8 +23,10 @@ const chart = computed(() => {
 
 watchDebounced(
 	() => chart.value?.doc.config.order_by,
-	() => chart.value?.refresh(),
-	{
+	() => {
+        const consolidatedFilters = Object.values(dashboard.filters).flat() as FilterArgs[];
```

Comment:
Thanks for pointing out the issue!

Filters are applied at the query level, and it may not necessarily be the same query that the chart is based on. Therefore, it's better to handle this in the same manner as during a dashboard refresh.